### DISCUSSION TITLE
Model-Viewer usage?

### DISCUSSION THREAD

**Discussion by lehoangphu** [google/model-viewer#3379] [Q&A]
Do we know of the usage of real-life production page? Like I know there are ~1.6k github projects depend on model-viewer project, but it would be nice to know about the usage and maybe a list of awesome/showcase websites that leverage Model-Viewer to its full potential?

**Comment by elalish** [comment]
I agree that this would be great! Unfortunately I was told Google does not like to create brand-promoting pages like that. Also, it's tough to keep links up to date since cool websites tend to change and move pretty often. Honestly, even I have trouble figuring out all of where MV is used, but our crawler shows me that it's quite broadly deployed, both in the sense of major brands, and the long tail, and is growing steadily. 

I always encourage users who do anything awesome with MV to mention us on [Twitter](https://twitter.com/modelviewer) (always happy to retweet!), start a [show-and-tell](https://github.com/google/model-viewer/discussions/3052) discussion here on Github, or even contribute an [example](https://modelviewer.dev/examples/annotations/index.html#cameraViews) to our dev pages.
